lib/apruve/resources/corporate_account.rb

```diff
@@ -17,5 +17,10 @@ def self.find_all(merchant_id)
       response = Apruve.get("merchants/#{merchant_id}/corporate_accounts")
       response.body.map { |ca| CorporateAccount.new(ca.empty? ? {} : ca) }
     end
+
+    def self.find_by_uuid(merchant_id, uuid)
```

Comment:
I'm wondering if this should just be self.find, using the uuid as the id, like we do for the other resources.

Reply:
Yeah, that is what makes sense idiomatically for sure. The reason I decided not to was to not break anything that uses this gem and the `find` method.

Reply:
We can change it for sure, we just need to make sure that we update anything that uses it already without the uuid

Reply:
Ideally the existing find method would be refactored to find_by_merchant. But I agree that it might break some things.